enhancements/monitoring/monitoring-stack-operator.md

```diff
@@ -0,0 +1,490 @@
+---
+title: Monitoring for Red Hat Managed Services
+authors:
+- @bwplotka
+- @fpetkovski
+reviewers:
+- @alanmoran
+- @smarterclayton
+- @kbsingh
+- @pb82
+- @simonpasquier
+approvers:
+- TBD
+creation-date: 2021.08.12
+last-updated: 2021.08.12
+status: implementable
+---
+
+# Monitoring for Red Hat Managed Services
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [X] Design details are appropriately documented from clear requirements
+- [X] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [X] User-facing documentation is created in [openshift-docs]([https://github.com/openshift/openshift-docs/](https://github.com/openshift/openshift-docs/)) - (there are non required)
+
+## Summary
+
+This document proposes a solution and a delivery plan for scraping, querying and alerting on metrics related to Red Hat Managed Services running in OSD clusters. The proposed solution is not intended to complement the existing monitoring stack managed by the Cluster Monitoring Operator, nor used directly by Customers at this moment.
```

Comment:
In an addon perspective, which SRE persona would likely consume the alerting part? For example, platform SREs, addon SREs or app SREs etc? The reason for asking is that Addons may have their own monitoring stack.